gatsby-node.js

```diff
@@ -60,15 +60,7 @@ exports.createPages = async ({graphql, boundActionCreators}) => {
   allMarkdown.data.allMarkdownRemark.edges.forEach(edge => {
     const slug = edge.node.fields.slug;
 
-    if (slug === '/index.html') {
```

Comment:
No longer need to programatically create a page because the home page is now within the `pages` directory.

Reply:
I dig it.